governance-aepp/src/views/Poll.vue

```diff
@@ -327,6 +327,15 @@
         }).catch(console.error);
 
         await Promise.all([fetchBalance, fetchPollAddress, fetchPollState, fetchVotesState]);
+
+        aeternity.verifyPollContract(await fetchPollAddress).then(verifiedPoll => {
```

Comment:
we should be able to use `this.pollAddress` instead of `await fetchPollAddress` since we already await it

Reply:
maybe also cache the verification result?

Reply:
the await shouldn't make a difference for pollAddress, but you are right it is already resolved.

I'll add another issue about caching of the result.